Campaign/management/commands/StartNewCampaign.py

```diff
@@ -0,0 +1,264 @@
+"""
+Appraise evaluation framework
+
+See LICENSE for usage details
+"""
+from datetime import datetime
+from os import path
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.files import File
+
+from Campaign.management.commands.init_campaign import (
+    _create_context,
+    _init_campaign,
+)
+from Campaign.management.commands.validatecampaigndata import _validate_campaign_data
+from Campaign.management.commands.ProcessCampaignData import _process_campaign_data
+from Campaign.models import (
+    Campaign,
+    CampaignData,
+    CampaignTeam,
+    Market,
+    Metadata,
+)
+from Campaign.utils import (
+    _identify_super_users,
+    _load_campaign_manifest,
+    _process_market_and_metadata,
+)
+from EvalData.management.commands.UpdateEvalDataModels import _update_eval_data_models
+
+from Dashboard.utils import generate_confirmation_token
+
+# pylint: disable=C0111,C0330,E1101
+class Command(BaseCommand):
+    help = 'A single command for creating a new campaign based on manifest file'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'manifest_json',
+            metavar='manifest-json',
+            type=str,
+            help='Path to manifest file in JSON format.',
+        )
+
+        # TODO: support adding multiple batches
+        parser.add_argument(
+            '--batches-json',
+            type=str,
+            default=[],
+            metavar='JSON',
+            nargs='+',
+            help='List of paths to batches in JSON format.',

```

Comment:
A note for myself: mention that the order of batches should be the same as the order of TASKS_TO_ANNOTATORS in the manifest file to have markets and metadata assigned correctly.